openmls/src/extensions/test_extensions.rs

```diff
@@ -243,3 +247,96 @@ fn required_capabilities() {
     assert_eq!(ext, ext_decoded);
     assert_eq!(extension_bytes, encoded);
 }
+
+#[apply(ciphersuites_and_providers)]
+fn last_resort_extension(ciphersuite: Ciphersuite, provider: &impl OpenMlsProvider) {
+    let last_resort = Extension::LastResort(LastResortExtension::default());
+
+    // Build a KeyPackage with a last resort extension
+    let credential = Credential::new(b"Test".to_vec(), CredentialType::Basic).unwrap();
```

Comment:
Since we're using it for Bob below, let's use it as identity here too.

```suggestion
    let credential = Credential::new(b"Bob".to_vec(), CredentialType::Basic).unwrap();
```